cross-chain-atomic-swap-cordapp/src/main/kotlin/jp/co/layerx/cordage/crosschainatomicswap/Utils.kt

```diff
@@ -30,11 +30,11 @@ fun Party.ethAddress(): String {
             ethAddress.load(url.openStream())
         }
         val root = System.getProperty("user.dir")
-        val path = Paths.get("$root/cordapps/config/cross-chain-atomic-swap-cordapp-0.1.conf")
+        val path = Paths.get("$root/cordapps/config/config.conf")
         if (Files.isReadable(path)) {
             ethAddress.load(Files.newInputStream(path))
         }
     }
 
-    return ethAddress.getProperty(this.name.toString(), "0x0")
+    return ethAddress.getProperty(this.name.organisation, ethAddress.getProperty(this.name.toString(), "0x0"))
```

Comment:
I think it is enough to write:
```
return ethAddress.getProperty(this.name.organisation, "0x0"))
```

because there no full name in the config. Is this right?

Reply:
Done